pages/create/index.page.js

```diff
@@ -1,72 +1,113 @@
+import { number, object, string, date, ref } from "yup";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useForm } from "react-hook-form";
+
 import Button from "../../components/button/button";
 import Input from "../../components/input/input";
 import Select from "../../components/select/select";
-import { Container, Form, Label, Title } from "./create.styles";
-import { useForm } from "react-hook-form";
-import { number, object, string, date, ref } from "yup";
-import { yupResolver } from "@hookform/resolvers/yup";
+
+import { Container, Form, Title } from "./create.styles";
 
 const Create = () => {
   const validationSchema = object().shape({
-    projectName: string().required(),
+    title: string().required(),
     subtitle: string().required(),
-    category: object().required(),
-    description: string().required(),
-    currency: object().required(),
+    story: string().required(),
+    token: string().required(),
     fundGoal: number().required(),
     startDate: date()
       .required()
-      .min(new Date(), "Created date can not be future"),
+      .max(new Date(), "Created date can not be future"),
     endDate: date()
       .required()
       .min(ref("startDate"), "End date should be after start date"),
   });
 
-  const {
-    handleSubmit,
-    reset,
-    formState: { errors },
-  } = useForm({
+  const formOptions = {
     resolver: yupResolver(validationSchema),
     defaultValues: {},
     mode: "onChange",
-  });
+  };
+  const { handleSubmit, reset, register, formState } = useForm(formOptions);
+  const { errors } = formState;
 
   const onSubmit = (data) => {
-    console.log("POST", data);
+    console.log("POST:", data);
+
+    // The testData was copied from the Postman requests:
+    const nextMinute = Date.now() + 1000 * 60;
+    const nextHour = Date.now() + 1000 * 60 * 60;
+    const testData = {
+      title: "My campaign",
+      subtitle: "An amazing campaign",
+      story: "This is the long short story: We need the money",
+      startDate: new Date(nextMinute),
+      endDate: new Date(nextHour),
+      image: "image.png",
+      video: "video.mp4",
+      goal: [
+        {
+          token: "637e538f6e0eb6be40e2790f",
+          amount: "100",
+        },
+      ],
+    };
+
+    fetch("http://localhost:10000/api/v1/campaigns", {
```

Comment:
We need to move this to a `apiClient` class that already knows the API url, otherwise we need to provide that URL on each fetch call 

Or at least use a environment variable, `localhost:1000` its hardcoded